tools/cvm-image-rewriter/plugins/08-ccnp-uds-directory-permission/README.md

```diff
@@ -0,0 +1,3 @@
+# CCNP UDS directory permission
```

Comment:
Same comment as above. If users don't need to customize why give the option and confuse them?